assets/components/organisms/table/table.scss

```diff
@@ -0,0 +1,189 @@
+@charset 'utf-8';
+
+.table {
+  thead {
+    th {
+      border-bottom: (3 * $table-border-width) solid $table-border-color;
+      padding: $table-cell-padding-sm $table-cell-padding-sm;
+    }
+  }
+
+  tbody,
+  tfoot {
+    color: $black;
+
+    td {
+      vertical-align: top;
+      padding: $table-cell-padding $table-cell-padding-sm;
+      border-bottom: $table-border-width solid $table-border-color-light;
+
+      @include media-breakpoint-down(sm) {
+        &:first-child {
+          margin-top: $table-cell-padding;
+        }
+
+        &:last-child {
+          margin-bottom: $table-cell-padding;
+        }
+      }
+
+      .tablesaw-cell-label {
+        display: inline-block;
+        min-width: 33%;
+        color: $body-color;
+      }
+    }
+  }
+
+  tfoot {
+    td {
```

Comment:
unnecessary nesting

Reply:
What do you mean? A class should be created instead?